CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ and this project adheres to
   set, it won't trigger the lockup condition. Calling the ioctl for guests that
   don't use kvmclock will fail. These failures are not fatal. We log the failure
   and increase the `vcpu.kvmclock_ctrl_fails` metric.
+- [#4869](https://github.com/firecracker-microvm/firecracker/pull/4869): Added
+  support for Aarch64 systems which feature CPU caches with a number of sets
+  higher than `u16::MAX`.
```

Comment:
nit: the  applicable hardware is only mentioned in the PR and none of the commits.

Reply:
I did that in purpose. My reasoning was, 1. it's not a platform we officially support and 2. it doesn't solve the problem for that particular platform only; it's a fix for any platform with a number of cache sets bigger than `u16::MAX`. WDYT?